testSuite/scripts/test_file_copy.py

```diff
@@ -0,0 +1,65 @@
+import os
+import unittest
+
+import utility as util
+
+
+class File_Copy_User_Scenario(unittest.TestCase):
+
+    def test_copy_single_file_to_file(self):
+        content_file_name_src = "test_1kb_file_copy_src.txt"
+        content_file_path_src = util.create_test_file(content_file_name_src, 1024)
+        content_file_name_dst = "test_1kb_file_copy_dst.txt"
+
+        # create source file of size 1KB.
+        remote_src_file_path = util.get_resource_sas_from_share(content_file_name_src)
+        remote_dst_file_path = util.get_resource_sas_from_share(content_file_name_dst)
+        result = util.Command("cp").add_arguments(content_file_path_src).add_arguments(remote_src_file_path). \
+            add_flags("log-level", "info").execute_azcopy_copy_command()
+        self.assertTrue(result)
+
+        # perform the single file copy using azcopy.
+        result = util.Command("copy").add_arguments(remote_src_file_path).add_arguments(remote_dst_file_path). \
+            add_flags("log-level", "info").execute_azcopy_copy_command()
+        self.assertTrue(result)
+
+        # verifying the copy worked, both remote source and destination should be identical to the local source
+        result = util.Command("testFile").add_arguments(content_file_path_src).add_arguments(
+            remote_src_file_path).execute_azcopy_verify()
+        self.assertTrue(result)
+        result = util.Command("testFile").add_arguments(content_file_path_src).add_arguments(
+            remote_dst_file_path).execute_azcopy_verify()
+        self.assertTrue(result)
+
+    def test_copy_entire_dir_to_dir(self):
```

Comment:
Consider adding a test using the trailing wildcard atop directory to directory

Reply:
Updated the entire_dir_to_dir to use /* instead of --strip-top-dir which was being used.